R/lnormdm.R

```diff
@@ -0,0 +1,79 @@
+#' Lognormal Data Maker : lnormdm
+#'
+#' A sandbox to create and plot lognormal data.
+#' Enter any number of treatment groups, their names, their respective log means, and their log standard deviations.
+#' Treatment groups can be uneven sizes.
+#' k must correspond with the number of values in names, meanlog, and sdlog.
+#'
+#' @param k is the number of treatment groups.
+#' @param names are the names of the treatment groups.
```

Comment:
should read "a vector of character strings of length k defining the treatment groups"

Reply:
this same "a vector..." convention should be used for all of the lnormdm parameters